api/tests/certification/course/integration/infrastructure/repositories/sco-certification-candidate-repository_test.js

```diff
@@ -124,6 +124,14 @@ describe('Certification | Course | Integration | Repository | SCOCertificationCa
         sessionId,
         organizationLearnerId: yetAnotherOrganizationLearnerId,
       }).id;
+
+      databaseBuilder.factory.buildCertificationCandidate({
```

Comment:
```suggestion
      // This user has taken a more recent session, only the last one will be retrieved
      databaseBuilder.factory.buildCertificationCandidate({
```
Ou à la place du commentaire, faire apparaître explicitement que ce candidat n'est pas le plus récent 🤔

Reply:
C'est explicité par la ligne 131 ```sessionId: sessionInThePastId```, mais on pourrait changer la variable par ```sessionId: olderSessionIdThatShouldNotBeInResult```

Reply:
Ou Stocker le candidatId dans une variable explicite et faire un expect avec un not include :thinking:

Reply:
Ca me semble explicite avec le nom de la session